AirCasting/CreateSessionViews/ConfirmCreatingSessionView.swift

```diff
@@ -92,8 +107,13 @@ struct ConfirmCreatingSessionView: View {
                 Text(Strings.ConfirmCreatingSessionView.startRecording)
                     .bold()
             })
-            .buttonStyle(BlueButtonStyle())
-        }.padding()
+                .buttonStyle(BlueButtonStyle())
+        }.onAppear(perform: {
+            let lat = locationTracker.googleLocation[0].location.latitude
```

Comment:
is it guaranteed that we'll have at least one location here so we don't crash on `[0]` ?

Reply:
It is not in the code anymore 😄